src/model/network.hpp

```diff
@@ -40,6 +40,13 @@ namespace model
 //               Network Specs                //
 //--------------------------------------------//
 
+enum ConnectionType : int {
+    UNUSED = 0,
+    RF = 1, // Read-Fill
+    UD = 2, // Update-Drain
+    RFUD = 3
+};
+
```

Comment:
Recommend expanding these acronyms to: Unused, ReadFill, UpdateDrain, ReadFillUpdateDrain